### ISSUE TITLE
Accessibility issue -  button-name

### ISSUE THREAD

**Issue by spacedmonkey** [ampproject/amp-wp#3307]
When running the automated accessibility tests in #3294 the follow "button-name" errors where found with story editor.  

```
  Rule: "button-name" (Buttons must have discernible text)
    Help: https://dequeuniversity.com/rules/axe/3.3/button-name?application=axe-puppeteer
    Affected Nodes:
      .components-toolbar:nth-child(2) > div > .has-text.components-icon-button.components-button
        Fix ANY of the following:
        - Element does not have inner text that is visible to screen readers
        - aria-label attribute does not exist or is empty
        - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        - Element's default semantics were not overridden with role="presentation"
        - Element's default semantics were not overridden with role="none"
        - Element has no title attribute or the title attribute is empty
      .components-toolbar:nth-child(3) > div > .has-text.components-icon-button.components-button
        Fix ANY of the following:
        - Element does not have inner text that is visible to screen readers
        - aria-label attribute does not exist or is empty
        - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        - Element's default semantics were not overridden with role="presentation"
        - Element's default semantics were not overridden with role="none"
        - Element has no title attribute or the title attribute is empty
   ```

**Comment by jauyong** [comment]
Hey @jasti , I wasn't sure what AC to put for this ticket.
Would something like "address the button-name errors found by the automated accessibility test" suffice? Or I could list out the things it says to fix as AC?
